src/components/table/draggable-table-cell/draggable-table-cell.scss

```diff
@@ -0,0 +1,16 @@
+.draggable-table-cell {
+  width: 21px;
```

Comment:
Not sure - presumably something to do with the width of the icon used for the drag handle?

@andrewjtait may know more?